client/src/components/HistoryViewer/HistoryViewerVersionList.js

```diff
@@ -28,12 +32,13 @@ class HistoryViewerVersionList extends PureComponent {
    * @returns {boolean}
    */
   isVersionActive(version) {
-    const { isActive, compare } = this.props;
-    if (isActive) {
-      return true;
-    }
+      const { currentVersion, compare, compare: { versionFrom, versionTo } } = this.props;
```

Comment:
I found this method a little confusing to skim read. Perhaps the doc block could be updated to include some notes about when it is true for compare mode.

The variables also smell like they could be their own methods to me - `isCurrent()`, `isCompareFrom()` etc.

I'm thinking something like this:

```js
// Check for a single selected version
if (this.isSelected(version)) {
  return true;
}

// Check if we're comparing between two versions
if (this.isSelectedForCompare(version)) {
  return true;
}

return false;
```

```js
isSelected(version) {
  return this.props.currentVersion && this.props.currentVersion.Version = version.Version;
}

isSelectedForCompare(version) {
  const { compare: { versionFrom, versionTo } } = this.props;

  return (versionFrom || versionTo) &&
    (versionFrom.Version === version.Version || versionTo.Version === version.Version);
}
```

Thoughts?

Reply:
I added more betterer docblock :>